### ISSUE TITLE
GitHub's new search bar quick jump

### ISSUE THREAD

**Issue by Kerumen** [algolia/github-awesome-autocomplete#82]
GitHub just rolled-out a new feature for its search bar:

<img width="382" alt="screen shot 2018-05-24 at 18 04 21" src="https://user-images.githubusercontent.com/5436545/40497694-2c8f876e-5f7d-11e8-9c74-ab55ba82d558.png">

Which leads to interferences with this extension:

<img width="431" alt="screen shot 2018-05-24 at 18 05 18" src="https://user-images.githubusercontent.com/5436545/40497701-324f50e4-5f7d-11e8-962f-d85cbd5cb329.png">

**Comment by rayrutjes** [comment]
Yes :/ we will need to do an update to take that feature into account.

**Comment by drewswaycool** [comment]
Why is this closed? It is still an issue and is unrelated to stylish themes.

**Comment by Haroenv** [comment]
This isn’t closed, still an open issue

**Comment by macintacos** [comment]
Just wanted to note that this is still a problem; it makes this extension basically unusable because GitHub defaults to using the value in its own searchbar instead of the one in this extension. Workaround for now is to just use the mouse to select the option brought up in the search, like an animal (/s)

**Comment by rayrutjes** [comment]
Thanks for pushing @macintacos , I'll check internally who could have some time to work on the extension.
